data/lang/module-combat/en.json

```diff
@@ -271,6 +271,18 @@
     "description": "Name of mission type. Derived from Fighter Sweep. An aggressive mission to hunt down and destroy every enemy fighter in the allotted area.",
     "message": "Area Sweep"
   },
+  "MISSION_TYPE_CONVO_1": {
+    "description": "Name of mission type, used in conversation",
+    "message": "a reconnaissance"
+  },
+  "MISSION_TYPE_CONVO_2": {
+    "description": "Name of mission type, used in conversation",
+    "message": "an armed reconnaissance"
```

Comment:
I wonder if this makes it hard to translate, when it's a fraction of a sentence. Perhaps description could mention which other translation variable makes use of it, and/or provide example sentence with surrounding context?